test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -32,9 +32,9 @@ public void emptyInput_returnsIncorrect() {
     }
 
     @Test
-    public void unknownCommandWord_returnsHelp() {
+    public void unknownCommandWord_returnsIncorrectCommand() {
         final String input = "unknowncommandword arguments arguments";
-        parseAndAssertCommandType(input, HelpCommand.class);
+        parseAndAssertCommandType(input, IncorrectCommand.class);
     }
```

Comment:
These tests do not value-add anything. How about tests that actually test if your suggested command feature worked?